webapp/loggedin/surveyViews/categoryEdit/actions.js

```diff
@@ -67,6 +69,27 @@ export const createCategory = () => async (dispatch, getState) => {
   return category
 }
 
+export const uploadCategory = (categoryUuid, file) => async (dispatch, getState) => {
+  const formData = new FormData()
+  formData.append('file', file)
+
+  const config = {
+    headers: {
+      'content-type': 'multipart/form-data'
+    }
+  }
+
```

Comment:
You don't need to set the content-type header, axios does it himself when using FormData